examples/scripts/dpo_visual.py

```diff
@@ -16,12 +16,13 @@
 accelerate launch examples/scripts/dpo_visual.py \
     --dataset_name HuggingFaceH4/rlaif-v_formatted \
     --model_name_or_path HuggingFaceM4/idefics2-8b \
-    --per_device_train_batch_size 1 \
-    --gradient_accumulation_steps 16 \
+    --per_device_train_batch_size 2 \
+    --gradient_accumulation_steps 32 \
     --dataset_num_proc 32 \
     --output_dir dpo_idefics_rlaif-v \
     --bf16 \
     --torch_dtype bfloat16 \
+    --gradient_checkpointing \
```

Comment:
Allows the training to fit in the memory without the need to resize images. (resizing has been removed as well, see under)